liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/PackageServerMojo.java

```diff
@@ -169,6 +168,17 @@ private String getPackageFileType() throws MojoFailureException {
             if (packageType != null && !packageType.equals("zip")) {
                 log.info(packageType + " not supported. Defaulting to 'zip'");
             }
+
+            // Check for `runnable` in `include` for deprecation before completely removing it in favor of `jar` `packageType`
+            if (include.contains("runnable")) {
```

Comment:
I think include could be null, in which case you would get a NPE here.

Reply:
That problem actually came up in a few tests. Fixed